src/task.c

```diff
@@ -913,6 +913,7 @@ CFI_NORETURN
 #endif
 
     ct->started = 1;
+    JL_PROBE_RT_START_TASK(ct, ptls);
```

Comment:
Should this not also be `RUN_TASK`? Would be good to add documentation to what each probe means.

Reply:
Ok, switched back to `RUN_TASK`, and confirmed that it works with BPFnative (with some changes to support multiple probes). Will add some docs to the devdocs.